ext_emconf.php

```diff
@@ -13,12 +13,12 @@
 $EM_CONF[$_EXTKEY] = [
     'title' => 'Kitodo.Presentation',
     'description' => 'Base plugins, modules, services and API of the Digital Library Framework. It is part of the community-based Kitodo Digitization Suite.',
-    'version' => '3.0.0',
+    'version' => '3.0.1',
```

Comment:
Shouldn't we rise to 3.1.0 already? To avoid problem with next release?

Reply:
done.